x-pack/docs/en/security/securing-communications/update-tls-certificates.asciidoc

```diff
@@ -0,0 +1,784 @@
+[[update-node-certs]]
+== Updating node security certificates
+You need to update your TLS certificates if your current node certificates might
+expire soon, you're adding new nodes to your secured cluster, or security
+breach has broken the trust of your certificate chain. In these instances where
+<<update-node-certs-same,your CA and key remains the same>>, you can sign new
+certificates using the original Certificate Authority (CA) that you used to sign your existing certificates.
+
+If you have to trust a new CA, you need to
+<<update-node-certs-different,generate a new CA certificate and key>> for each
+node that is signed by the new CA certificate and instruct your nodes to trust
+this certificate chain.
+
+Depending on which certificates are expiring, you might need to update the
+certificates for the transport layer, the HTTP layer, or both. 
+
+Regardless of the scenario, {es} monitors the SSL resources for updates
+on a five-second interval, which is the default. You can just copy the new
+certificate and key files (or keystore) into the {es} configuration directory
+and your nodes will detect the changes and reload the keys and certificates. 
+
+Because {es} doesn't reload the `elasticsearch.yml` configuration,
+you must use *the same file names* so that {es} refreshes the security
+certificates and keys.
```

Comment:
I'll change to read:
>Because {es} doesn't reload the `elasticsearch.yml` configuration, you must use *the same file names* if you want to take advantage of automatic certificate and key reloading.